### PR TITLE
[WIP] Create consumptions migration

### PR DESCRIPTION
**Pull request by aljesusg**
Describe the rationale and use case for this pull request.  Provide any background, examples, and images that provide further information to accurately describe what it is that you are adding to the repo.  Add subsections as necessary to organize and feel free to link and reference other PRs as necessary, but also include them in the links section below as a quick reference.

Guidelines:
* Keep Pull Request titles short and to the point, ideally under 72 characters
* Provide as much context/info in the description as necessary to get the reviewer up to the same domain knowledge level as yourself
* Keep code changes as short as possible and implementing a single feature/fix/refactoring, when possible

Links [Optional]
----------------

* http://documentation.for/library/that/I/am/adding
* [PR relevant issue or pull_request](#123)

Steps for Testing/QA [Optional]
-------------------------------

If there are any manual steps that you would like the reviewer(s) to take to verify your changes, please describe in detail the steps to reproduce the features added by the pull request, or the bug before and after the change.

### CODE REVIEW

**Comment by sergio-ocon** [comment]
Peazo de descripción detallada...

File: db/migrate/20170222180722_create_consumptions.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      t.bigint   :configuration_consumption
+      t.text     :data
+      t.datetime :t_ini
+      t.datetime :t_end
+      t.datetime :updated_at
+      t.datetime :created_at
+      t.bigint   :id_object
+      t.string   :type_object
+    end
+  end
```
Comment (sergio-ocon) [inline db/migrate/20170222180722_create_consumptions.rb]: There is no index, if buckets are going to point to the ID, that should be an index

File: db/migrate/20170222180722_create_consumptions.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class CreateConsumptions < ActiveRecord::Migration[5.0]
+  def up
+    create_table :consumptions do |t|
+      t.bigint   :configuration_consumption
+      t.text     :data
```
Comment (aljesusg) [inline db/migrate/20170222180722_create_consumptions.rb]: I' ll make validations in model also

File: db/migrate/20170222180722_create_consumptions.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+class CreateConsumptions < ActiveRecord::Migration[5.0]
+  def up
+    create_table :consumptions do |t|
+      t.bigint   :configuration_consumption
```
Comment (aljesusg) [inline db/migrate/20170222180722_create_consumptions.rb]: write fail

File: db/migrate/20170222180722_create_consumptions.rb (hunk 2 above)
Comment (sergio-ocon) [inline db/migrate/20170222180722_create_consumptions.rb]: 👍

**Pull request closed without merging**